lighthouse-cli/test/smokehouse/test-definitions/dobetterweb/dbw-expectations.js

```diff
@@ -12,6 +12,7 @@
 const expectations = [
   {
     artifacts: {
+      HostUserAgent: /Chrome\/83/,
```

Comment:
will delete if it passes in appveyor

Reply:
Ok, the HostUserAgent passed the test, so looks like this is working. https://ci.appveyor.com/project/paulirish/lighthouse/builds/34028162#L481